File: C#/Program14.cs
/*.Write a program which accept number from user and display its multiplication of 
factors
Input : 12
Output : 144 (1 * 2 * 3 * 4 * 6)
Input : 13
Output : 1 (1)
Input : 10
Output : 10 (1 * 2 * 5)
.*/


using System;


class Demo{
    public int MultiplicationOfFactor(int iNo){

        int iMulti = 1;

        for(int i = 1; i<iNo; i++)
        {
            if(iNo % i == 0)
            iMulti = iMulti * i;
        }

        return iMulti;
    }
}

class program14{
    public static void Main(string[] args){
       
       int iNo = Convert.ToInt32(Console.ReadLine());

       Demo dobj = new Demo();

       int iRet = dobj.MultiplicationOfFactor(iNo);

       Console.WriteLine(iRet);

    }
}

File: JAVA/Program14.java
/*

Write a C++ program which accept range from user and display all even numbers in
between that range.

Input : 23 35
Output : 24 26 28 30 32 34

*/



import java.util.*;

class Demo14
{
     public void DisplayEven(int iStart,int iEnd)
     {
 
         int i = iStart;

         System.out.print("\n");

         while(i <= iEnd)
         {
         	 if((i % 2 ) == 0)
         	 {

         	   System.out.print(i+"\t");

         	}

         	i++;
         }




     }




}



class Program14
{
	public static void main(String[] args) 
	{
	
      Scanner sobj = new Scanner(System.in);

      Demo14 dobj = new Demo14();

      int iVal1 = 0;
      int iVal2 = 0;

      System.out.print("\nEnter Starting Point : ");
      iVal1 = sobj.nextInt();

      System.out.print("\nEnter Ending Point : ");
      iVal2 = sobj.nextInt();

      dobj.DisplayEven(iVal1,iVal2);

      System.out.print("\n");



	}
}